documentation/src/main/asciidoc/topics/ref_rest_clusters.adoc

```diff
@@ -161,3 +162,39 @@ Content-Length: 7353
 <zip-bytes>
 --5ec9bc07-f069-4662-a535-46069afeda32--
 ----
+
+[id='rest_v2_restore_cluster_list']
+= Listing Restores
+Retrieve the names of all restore requests that are in progress, completed, or
+failed.
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /rest/v2/cluster/restores
+----
+
+[id='rest_v2_restore_cluster_head']
+= Checking Restore Progress
+Verify that a restore operation is complete. A `200` response indicates the
+restore operation has completed. A `202` response indicates the backup operation is
+in progress.
+
+[source,options="nowrap",subs=attributes+]
+----
+HEAD /rest/v2/cluster/restores/{restoreName}
+----
+
+[id='rest_v2_restore_cluster_delete']
+= Deleting Restore Requests
+Remove restore requests from the server. This does not remove any restored content, it
+simply removes all meta information associated with a given request, therefore allowing
+subsequent restore requests to re-use a past request name.
```

Comment:
Same as the previous comment. Maybe we should change the heading as well? 

= Deleting Request Metadata 
Remove metadata for restore requests from the server. This action removes all metadata associated with restore requests but does not delete any restored content. If you delete the request metadata, you can use the request name to perform subsequent restore operations.

Reply:
I like it, I have used your suggestion but with the title changed to "Deleting Restore Metadata"